CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.13.0 / 2019-10-04
+
+* [BUGFIX] UI: fix for [CVE-2019-10215](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-10215). #6098
```

Comment:
SECURITY/BUGFIX. I'd also mention what it's fixing here, rather than entirely relying on a 3rd party site.

Reply:
yep , updated.

Reply:
CVE listings are quite stable, I wouldn't worry about it.